### ISSUE TITLE
Add cron parameter to the .job file

### ISSUE THREAD

**Issue by marknorkin** [azkaban/azkaban#1989]
On our project we are using Azkaban and we need to schedule Apache Spark job on some schedule. 

Looking at documentation https://azkaban.github.io/azkaban/docs/latest/ and https://github.com/azkaban/azkaban/wiki/Cron-Based-Schedule there is only option to configure job through UI or by making HTTP request.

Is it possible to have inside _.job_ file something like:
```
type=command
user.to.proxy=someuser
cron=* * * * *
command=spark-submit --class com.mycompany.MyApp ...
```

**Comment by marknorkin** [comment]
Our DevOps team currently also looking at extending the file format, providing some wrappers around it etc. In whatever approach will be chosen it is an extra work.

The benefit of this feature is that it will have clear usage, no customized instruments built on top of it which would require maintenance.

What issues do you see here, can you elaborate, please ?

**Comment by marknorkin** [comment]
Yes, sorry if I wasn't clear. This is about feature request. Initially I just wasn't sure if this was already implemented, which would mean that I just missed it in the documentation.

**Comment by HappyRay** [comment]
@chengren311 added support for this feature on top of the flow 2.0 design @jamiesjc did. @chengren311 can share more details.

**Comment by abhinav-29** [comment]
This would be a useful feature.
